test/performance/long-running-tests/README.md

```diff
@@ -0,0 +1,174 @@
+
+   ```bash 
+   git clone -b release-1.31 https://github.com/kubernetes/perf-tests.git
+   ```
+
+   Set the variable `CL2_DIR` to the path of the subdirectory `clusterloader2/` of the cloned repo. For example: 
+
+   ```bash 
+   export CL2_DIR=$HOME/perf-tests/clusterloader2
+   ```
+
+2. Configure clusterloader2 to generate KS performance workloads:
+
+   Starting from a local directory containing this KubeStellar git repo, run the following script based on your cluster environment:
+
+   a) cd into the `test/performance/common` directory from your local copy of the KubeStellar repo, for example:
+
+   ```bash 
+   cd .$HOME/kubestellar/test/performance/common
+   ```
+
+   b) set-up your environment:
+
+   ```bash 
+   ./setup-clusterloader2.sh --exp l
+   ```
+
+3. Configure the parameters of your workload:  
+
+   a) cd into the load configuration directory
+
+   ```bash 
+   cd  $CL2_DIR/testing/load/
+   ```
+  
+   b) configure the parameters to create the workload traffic (e.g., RandomizedLoad, SteppedLoad, Sequence, etc.)
+   
+   ```bash 
+   vi long-duration-exp-config.yaml
+   ``` 
+
+   More specifically, configure the following parameter: 
+
+   - numberOfWorkloadObjects: total number of objects to be created in the experiment `perf-test` namespace (*default value: "10"*)
+  
+   To learn more about clusterloader2 tuningSet, see the following: [tuningSets](https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/design.md#tuning-set) and [configurations](https://github.com/kubernetes/perf-tests/blob/fac2a5eec96fab76a4bc4858795df4544b729b0b/clusterloader2/api/types.go#L249).
+
+
+4. Deploy your first workload:
+
+   First, use a kubeconfig that includes contexts for WDS spaces and set its current context to a target WDS space (e.g., `wds1`):
+
+   ```bash 
+   kubectl config use-context wds1
+   ```
+
+   Output:
+   ```console 
+   Switched to context "wds1".
+   ```
+   
+   Optionally, check the kubeconfig contexts: 
+   ```bash 
+   kubectl config get-contexts
+   ```
+   
+   Output:
+   ```console 
+   CURRENT   NAME            CLUSTER         AUTHINFO        NAMESPACE
+             cluster1        kind-cluster1   kind-cluster1   
+             cluster2        kind-cluster2   kind-cluster2   
+             its1            its1-cluster    its1-admin      
+             kind-kubeflex   kind-kubeflex   kind-kubeflex   
+   *         wds1            wds1-cluster    wds1-admin      default
+             wds2            wds2-cluster    wds2-admin      default
+   ```
+
+   Second, create the binding Policy and namespace for your experiment using the following commands:
+
+   ```bash 
+   cd $CL2_DIR/testing/load/
+   kubectl apply -f long-duration-exp-setup.yaml
+   ```
+
+   Lastly, in the following set ``--kubeconfig`` flag to the path of the kubeconfig file used above and use the newly added Kubestellar provider (`--provider=ks`) and run the following command to create the workload:
```

Comment:
The wording here does not line up with the example command shown below, which says `--kubeconfig=<path>/wds-kubeconfig`. This example command seems to suppose that there is a kubeconfig file for each space. But that is not what the other commands suppose. For example, the `kubectl config use-context wds1` shown above --- which assumes that no kubeconfig file needs to be specified, equivalently that the usual rules about `$KUBECONFIG` and `~/.kube/config` will lead to the right kubeconfig file.

Given the `kubectl config use-context wds1` above, why does the `go run cmd/clusterloader.go` command need to have a kubeconfig explicitly specified?

Same issues appear in the short-running doc.

Reply:
When I tested, I discovered that clusterloader insists on the `--kubeconfig` flag:

```console
mspreitz@mjs13 clusterloader2 % go run cmd/clusterloader.go --testconfig=./testing/load/long-duration-exp-config.yaml  --provider=ks --v=2                                  
I0114 01:42:40.623129   26741 dns_performance-k8s-hostnames.go:58] Registering measurement: DNS Performance for K8s Hostnames
I0114 01:42:40.623316   26741 network_performance_measurement.go:90] Registering Network Performance Measurement
F0114 01:42:40.625412   26741 clusterloader.go:283] Parsing flags error: [no kubeconfig path specified when --run-from-cluster is unset]
I0114 01:42:40.625413   26741 clusterloader.go:267] Listening on 8000
exit status 1
```

Reply:
Yes! clusterloader requires the `--kubeconfig` flag indeed.